.github/workflows/build.yaml

```diff
@@ -15,12 +15,31 @@ jobs:
       - name: Setup Linux Build Dependencies
         if: ${{ runner.os == 'Linux' }}
         run: sudo apt-get install ninja-build
+
+      - name: Run apt update
+        if: runner.os == 'Linux'
+        run: sudo apt update
+
+      - name: Run apt update
+        if: runner.os == 'Linux'
+        run: sudo apt upgrade
```

Comment:
Is the system-wide package update needed? We have generally been leaving updating system packages to the source VM images and only pulling in updated dependencies specific to the build. (Follows [GH's customization instructions](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners/customizing-github-hosted-runners)) Running a full upgrade also takes more time.

Reply:
I'm not 100% sure as I've not tried. I can try later if it works and produces a SDL2 library the viewer can consume and create an OpenGL context with